tests/sniffer.py

```diff
@@ -59,6 +59,20 @@ def ieee_filter(x): return 'ieee1905' in x['_source']['layers']
         except BaseException as ex:
             return None
 
+    def _simplify_packets(self, packets):
+        def _simplify(x):
+            x['src_mac'] = x['_source']['layers']['eth']['eth.src']
```

Comment:
(nitpick) Instead of "simplifying", it would be nicer to convert to a brand new class Ieee1905Message. Or a namedtuple instead o a full class.